app/services/remove_nccc_email_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# rubocop:disable Layout/LineLength
+class RemoveNcccEmailService < ::WasteExemptionsEngine::BaseService
+
+  # https://rubular.com/r/LPHA79nxD5k2ff
+  # rubocop:disable Style/StringLiterals
+  WASTE_EXEMPTIONS_EMAIL_VARIATIONS = '^(w|a)[qaste]?\w{2,}(-?(e|-)|)(.?e)\w{7,}@(e|a|n)\w{5,17}(.|-)(ag?(ency|ancy|ncy)|gov).(gov.uk|uk)'
+
+  EXEMPTION_EMAIL_VARIATIONS = '^e(nquiries|xcemption?(s|)|xceptions|xemptions)@environment-agency.gov.uk'
+
+  NCCC_EMAIL_VARIATIONS = '^nccc(-?[cew]\w{4,}(-\w{10,}@|@)|@)enviro\w{4,}-agency.gov.uk'
+  # rubocop:enable Style/StringLiterals
+
+  def run
+    matching_registrations = find_email_variations(:applicant_email)
+
+    return unless matching_registrations.any?
+
+    Rails.logger.info "Clearing wex applicant email addresses: #{matching_registrations.pluck(:applicant_email)}"
+    matching_registrations.update_all(applicant_email: nil)
+
+    matching_registrations = find_email_variations(:contact_email)
+
+    return unless matching_registrations.any?
+
+    Rails.logger.info "Clearing wex contact email addresses: #{matching_registrations.pluck(:contact_email)}"
+    matching_registrations.update_all(contact_email: nil)
+  end
```

Comment:
So if there are no matches found for `:applicant_email` then the rest of the searches have no reason to run?

Reply:
This was another annoying rubocop error. I had them in if statements and rubocop disagreed.